packages/block-library/src/image/edit.native.js

```diff
@@ -357,9 +357,9 @@ export class ImageEdit extends React.Component {
 			},
 		];
 
-		const sizeOptions = map( imageSizes, ( { label, slug } ) => ( {
+		const sizeOptions = map( imageSizes, ( { name, slug } ) => ( {
```

Comment:
Updated to accommodate this change - https://github.com/WordPress/gutenberg/pull/19800